lib/wallet/src/Cardano/Wallet/Delegation.hs

```diff
@@ -289,11 +299,16 @@ quitStakePool netLayer db timeInterpreter = do
         , txDeposit = Just $ W.stakeKeyDeposit pp
         }
 
-guardQuit :: WalletDelegation -> Withdrawal -> Coin -> Either ErrCannotQuit ()
-guardQuit WalletDelegation{active,next} wdrl rewards = do
+guardQuit
+    :: WalletDelegation
+    -> Withdrawal
+    -> Coin
+    -> Bool
```

Comment:
```suggestion
```

Isn't this piece of information contained in the `WalletDelegation` argument? 🤔 

```hs
voting = case last_ of
   Voting{} → True
   DelegatingVoting{} → True
   _ → False
```

Reply:
Probably, but let's leave it like this for now — the argument is localized to the `Cardano.Wallet.Delegation` module now.